packages/utilities/psammead-locales/moment/.eslintrc.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
```

Comment:
Why not in base eslint file?

Reply:
Because I assume we want to maintain our use of dangling commas everywhere else than here, the moment locales are not transpiled before being run on the client however so need special treatment.

We could move away from dangling commas everywhere if we want but would be a major change.